internal/services/apimanagement/parse/api.go

```diff
@@ -44,7 +44,7 @@ func (id ApiId) ID() string {
 func ApiID(input string) (*ApiId, error) {
 	id, err := resourceids.ParseAzureResourceID(input)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("parsing %q as a Api ID: %+v", input, err)
```

Comment:
Can we cover `A` vs `An` cases for grammatical correctness?

Reply:
Done